### ISSUE TITLE
semi auto annotation function message Error : "point coordinate" is expected to be "number", but "object has been got.

### ISSUE THREAD

**Issue by sanghun-scalawox** [opencv/cvat#6692]
Hi, I wanted to execute semantic segmentation(deeplabv3 inference) function, but I got error like below image. 

![image](https://github.com/opencv/cvat/assets/79188784/ce2cea52-4d3f-4ab2-affe-87bb20df778e)
my reslult is list, and object of list is like this. {
                            "label": str,
                            "points": [ [x1,y1],[x2,y2] ...]
                            "mask": cvat_mask.tolist(),
                            "type": "mask"}

below code is inference method of ModelHandler class.
```    
    def infer(self, image, threshold):
        '''
            image(PIL Image)
        '''
        input_tensor = self.preprocess(image)
        input_batch = input_tensor.unsqueeze(0).to(self.device)
        output = self.model(input_batch)        
        preds = output.argmax(1).detach().cpu().numpy()
        results=[]
        for pred in preds:
            color_mask = ImageHelper.label2bgr(pred, self.colors)
            for class_index in range(1,len(self.colors)):
                label_class = ImageHelper.get_label(pred, class_index)
                if label_class is not None:
                    class_binary_mask = label_class==class_index
                    label_image = measure.label(class_binary_mask, connectivity=2)
                    for num,region in enumerate(measure.regionprops(label_image)):
                        cvat_mask=to_cvat_mask(region.bbox,class_binary_mask.astype(np.uint8))
                        results.append({
                            "label": self.labels[class_index],
                            "points": region.coords.tolist(), # [ [x1,y1],[x2,y2] ...]
                            "mask": cvat_mask.tolist(),
                            "type": "mask",
                        })
                        
        return results
```
and below code is to_cvat_mask method.
 ```
def to_cvat_mask(box, mask):
    min_row, min_col, max_row, max_col = box
    return mask[min_row:max_row,min_col:max_col]

```

it was nothing error message in function container's log.

what is wrong with my code?

i already saw issue #3620, but i cannot find the specified interface in cvat/apps/lambda_manager/views.py . Please Can you help me?

**Comment by sanghun-scalawox** [comment]
Thank you for your help. I refered to coco code. 
region.coord was not boundary coord, it was coord of region's all pixel. 
I'll close issue.
